src/makeCalculator.js

```diff
@@ -4,7 +4,32 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    operate: function (func, num) {
+      func.call(this, num);
+
+      return this;
+    },
+    add: function (a) {
+      this.result += a;
+    },
+    subtract: function (a) {
+      this.result -= a;
+    },
+    multiply: function (a) {
+      this.result *= a;
+    },
+    divide: function (a) {
+      this.result /= a;
```

Comment:
Consider adding a check to handle division by zero in the `divide` function to prevent potential runtime errors or unexpected results.